application/common/helpers/Utils.php

```diff
@@ -26,13 +26,14 @@ public static function getDatetime($timestamp = null)
     }
 
     /**
-     * @param integer|null $timestamp
+     * @param integer|string|null $timestamp time as unix timestamp, mysql datetime, or null for now
      * @return string
      */
     public static function getIso8601($timestamp = null)
     {
         $timestamp = $timestamp ?: time();
```

Comment:
An unlikely edge case, but this would make it impossible to get the ISO 8601 value for a unix timestamp of zero (0). It might be better to just go ahead and actually check `$timestamp !== null`.